Trellix/trellix-network-security/tests/malware_object.json

```
@@ -0,0 +1,72 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "trellix-network-security",
+        "dialect_uuid": "bae128bb-98c6-45f7-9763-aad3451821e5"
+      }
+    },
+    "message": "CEF:0|Trellix|MPS|10.0.0.992057|MO|malware-object|4|src=1.2.3.4 spt=49207 smac=6c:af:1a:fb:fe:a7 dst=5.6.7.8 dpt=80 dmac=00:78:db:db:96:f6 dvchost=cms-nx5600-1.eng.fireeye.com dvc=3.4.5.6 cn1Label=vlan cn1=0 cn2Label=sid cn2=8816733 cs1Label=sname cs1=Exploit.JAVA.CVE-2013-0422.FEC2 cs4Label=link cs4=https://cms-nx5600-1.eng.fireeye.com/detection/objects?uuid\\=860e5b30-5a8b-4159-8eb5-148ec3387e87 filePath=kentuckyautoexchange.com/tsh.jar rt=Sep 05 2023 16:28:42 UTC shost=dynamic-ip-adsl.viettel.vn fileHash=517f9835592fe08912c702c70219b20a externalId=8838994 act=notified devicePayloadId=860e5b30-5a8b-4159-8eb5-148ec3387e87 fileType=jar sproc=Java JDK JRE 7.13 fsize=13676 fname=tsh.jar flexString1Label=sha256sum flexString1=6e46b55feaeee973cfebabda18fa004b676a4be0919fd79bbad63f9f6a7a98f2 start=Sep 04 2023 11:26:23 UTC dvcmac=e3:e9:d0:5e:ba:8e"
+  },
+  "expected": {
+    "message": "CEF:0|Trellix|MPS|10.0.0.992057|MO|malware-object|4|src=1.2.3.4 spt=49207 smac=6c:af:1a:fb:fe:a7 dst=5.6.7.8 dpt=80 dmac=00:78:db:db:96:f6 dvchost=cms-nx5600-1.eng.fireeye.com dvc=3.4.5.6 cn1Label=vlan cn1=0 cn2Label=sid cn2=8816733 cs1Label=sname cs1=Exploit.JAVA.CVE-2013-0422.FEC2 cs4Label=link cs4=https://cms-nx5600-1.eng.fireeye.com/detection/objects?uuid\\=860e5b30-5a8b-4159-8eb5-148ec3387e87 filePath=kentuckyautoexchange.com/tsh.jar rt=Sep 05 2023 16:28:42 UTC shost=dynamic-ip-adsl.viettel.vn fileHash=517f9835592fe08912c702c70219b20a externalId=8838994 act=notified devicePayloadId=860e5b30-5a8b-4159-8eb5-148ec3387e87 fileType=jar sproc=Java JDK JRE 7.13 fsize=13676 fname=tsh.jar flexString1Label=sha256sum flexString1=6e46b55feaeee973cfebabda18fa004b676a4be0919fd79bbad63f9f6a7a98f2 start=Sep 04 2023 11:26:23 UTC dvcmac=e3:e9:d0:5e:ba:8e",
+    "event": {
+      "kind": "event",
+      "dataset": "malware-object",
+      "severity": 4,
+      "start": "2023-09-04T11:26:23Z",
+      "action": "notified",
+      "url": "https://cms-nx5600-1.eng.fireeye.com/detection/objects?uuid\\=860e5b30-5a8b-4159-8eb5-148ec3387e87",
+      "category": ["network"],
```

Could you update the parser to categorize `malware-object` event as `malware`?
```suggestion
      "category": ["malware"],
```